### PR TITLE
Ask for ticket type before selecting tickets

### PR DESCRIPTION
**Pull request by patrick91**

### CODE REVIEW

File: frontend/src/components/tickets-form/index.tsx [hunk 1]
```diff
@@ -9,36 +9,58 @@ import { SelectedProductsWithVariationsList } from "./selected-products-with-var
...
-    />
-  </React.Fragment>
-);
+}) => {
+  const ticketsToShow = tickets.filter(ticket => {
+    if (ticket.variations!.length > 0) {
+      return true;
+    }
+
+    if (isBusiness && ticket.name.toLowerCase().includes("business")) {
```
Comment (marcoacierno) [inline frontend/src/components/tickets-form/index.tsx]: I fear we do not have a better way to filter business and not business tickets :( @patrick91?

File: frontend/src/components/tickets-form/index.tsx [hunk 2]
```diff
@@ -9,36 +9,58 @@ import { SelectedProductsWithVariationsList } from "./selected-products-with-var
...
+
+  return (
+    <React.Fragment>
+      {ticketsToShow.map(ticket => (
+        <Box key={ticket.id}>
+          <ProductRow
+            quantity={
+              selectedProducts[ticket.id]
+                ? selectedProducts[ticket.id].length
+                : 0
```
Comment (marcoacierno) [inline frontend/src/components/tickets-form/index.tsx]: this is something I wrote do you think we can do `selectedProducts?.length ?? 0`?

**Comment by marcoacierno** [comment]
Looks fine to me! 

There is a typescript error:

```
src/components/tickets-page/index.tsx(43,7): error TS2322: Type 'string' is not assignable to type 'boolean'.
```

File: frontend/src/components/tickets-form/index.tsx (hunk 1 above)
Comment (patrick91) [reply frontend/src/components/tickets-form/index.tsx]: we could use categories (and have a flag on graphql, so we do this check on the backend)

**Pull request merged**